packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -259,12 +263,48 @@ const ReleaseDetailsBody = () => {
   const { formatMessage } = useIntl();
   const { releaseId } = useParams<{ releaseId: string }>();
   const [{ query }] = useQueryParams<GetReleaseActionsQueryParams>();
+  const toggleNotification = useNotification();
+  const { formatAPIError } = useAPIErrorHandler();
 
   const { isLoading, isFetching, isError, data } = useGetReleaseActionsQuery({
     ...query,
     releaseId,
   });
 
+  const [updateReleaseAction] = useUpdateReleaseActionMutation();
+
+  const handleChangeType = async (
+    e: React.ChangeEvent<HTMLInputElement>,
+    actionId: ReleaseAction['id']
+  ) => {
+    const response = await updateReleaseAction({
+      params: {
+        releaseId,
+        actionId,
+      },
+      body: {
+        type: e.target.value as ReleaseAction['type'],
+      },
+    });
+
+    if ('data' in response) {
+      // When the response returns an object with 'data', handle success
+      // In this case, we don't show anything if the request was successful
```

Comment:
Do we even need to check it then? Couldn't we just look for 'error' in response and handle that?